js/team.js

```diff
@@ -44,116 +44,92 @@ function _goBack() {
 	},
 	{
 		name: 'Minghao Ji',
-		program: '3rd Year Electrical Engineering',
+		program: '4th Year Electrical Engineering',
 		title: 'Technical Manager',
 		imgSrc: 'Minghao_Ji'
 	},
 	{
-		name: 'Tak Alguire',
-		program: '2nd Year Electrical Engineering',
-		title: 'Operations Manager',
-		imgSrc: 'Tak_Alguire'
-	},
-	{
 		name: 'Devon Copeland',
-		program: '3rd Year Mechatronics Engineering',
+		program: '4th Year Mechatronics Engineering',
 		title: 'Mechanical Manager',
 		imgSrc: 'Devon_Copeland'
 	},
 	{
 		name: 'Matthew Suski',
-		program: '2nd Year Mechanical Engineering',
+		program: '3rd Year Mechanical Engineering',
 		title: 'Mechanical Manager',
 		imgSrc: 'Matthew_Suski'
 	},
 	{
 		name: 'Ritu Shah',
-		program: '3rd Year Systems Design Engineering',
+		program: '4th Year Systems Design Engineering',
 		title: 'Chassis Lead',
 		imgSrc: 'Ritu_Shah'
 	},
 	{
-		name: 'Angela Chao',
-		program: '3rd Year Systems Design Engineering',
-		title: 'Chassis Lead',
-		imgSrc: 'Angela_Chao'
-	},
-	{
-		name: 'Kuldeep Panjwani',
-		program: '2nd Year Mechanical Engineering',
-		title: 'Body & Aerodynamics Lead',
-		imgSrc: 'Kuldeep_Panjwani'
-	},
-	{
 		name: 'Lauren Hankins',
 		program: 'Mechanical Engineering',
 		title: 'Brakes Lead',
 		imgSrc: 'Lauren_Hankins'
 	},
 	{
 		name: 'Robin Pearce',
-		program: '1st Year Mechatronics Engineering',
+		program: '2nd Year Mechatronics Engineering',
 		title: 'Driver Controls Lead',
 		imgSrc: 'Robin_Pearce'
 	},
 	{
 		name: 'Titus Chow',
-		program: '3rd Year Computer Science',
+		program: '4th Year Computer Science',
 		title: 'Electrical Manager',
 		imgSrc: 'Titus_Chow'
 	},
 	{
 		name: 'Calder Kitagawa',
-		program: '3rd Year Nanotechnology Engineering',
+		program: '4th Year Nanotechnology Engineering',
 		title: 'Electrical Manager',
 		imgSrc: 'Calder_Kitagawa'
 	},
 	{
-		name: 'Lucas Francisco Fryzek',
-		program: '3rd Year Computer Science',
-		title: 'Firmware Lead',
-		imgSrc: 'Lucas_Francisco'
-	},
-	{
 		name: 'Karl Ding',
-		program: '3rd Year Computer Science',
+		program: '4th Year Computer Science',
 		title: 'Firmware Lead',
 		imgSrc: 'Karl_Ding'
 	},
 	{
 		name: 'Kevin Chen',
-		program: '3rd Year Mechatronics Engineering',
+		program: '4th Year Mechatronics Engineering',
 		title: 'Hardware Design Lead',
 		imgSrc: 'Kevin_Chen'
 	},
 	{
 		name: 'Taiping Li',
-		program: '2nd Year Electrical Engineering',
+		program: '3rd Year Electrical Engineering',
 		title: 'Hardware Design Lead',
 		imgSrc: 'Taiping_Li'
 	},
 	{
 		name: 'Clarke Vandenhoven',
-		program: '1st Year Chemical Engineering',
+		program: '2nd Year Chemical Engineering',
 		title: 'Sponsorship Lead',
 		imgSrc: 'Clarke_Vandenhoven'
 	},
 	{
-		name: 'Iris Chang',
-		program: '2nd Year Systems Design Engineering',
-		title: 'Business Manager',
-		imgSrc: 'Iris_Chang'
-	},
-	{
 		name: 'Toni Li',
-		program: '2nd Year Management Engineering',
+		program: '3rd Year Management Engineering',
```

Comment:
Check with Toni cuz i don't think this information is correct

Reply:
@loicmurumba

Reply:
@STBTony I talked to Tak and he thinks we should remove the years, I agree too